### ISSUE TITLE
Merging with syntax-lang

### ISSUE THREAD

**Issue by jackfirth** [AlexKnauth/syntax-macro-lang#1]
Do you think it makes sense to have both the features from jackfirth/racket-syntax-lang and this one provided in the same library? Or to even provide `define-lang-syntax` at all?

**Comment by AlexKnauth** [comment]
I personally don't like the idea of a language being defined by a `define-lang-synatx` form like that. I'm not sure why. Partially because it's not a binding or a definition that can be required or provided. But that's just a general feeling I have.

**Comment by jackfirth** [comment]
I can see that. Perhaps using `define` in the name is a misnomer?

**Comment by AlexKnauth** [comment]
Yes, but also, I feel like it should be obvious that a language is defined by a module, whether a `.../lang/reader.rkt` module or a `(submod ... reader)` module. I actually prefer the `.../lang/reader.rkt` method because it's more obvious, from just looking at the directory, that it defines a #lang language. But having a special `syntax/macro-lang` language declared at the top of the main file is almost as obvious, so that's one reason why I would prefer that over a `define-lang-syntax`-like form somewhere in the body.
